tools/wheels/cibw_before_build_win.sh

```diff
@@ -11,4 +11,4 @@ python -m pip install -r requirements/openblas.txt
 python -c "import scipy_openblas32; print(scipy_openblas32.get_pkg_config())" > $PROJECT_DIR/scipy-openblas.pc
 
 # delvewheel is the equivalent of delocate/auditwheel for windows.
-python -m pip install delvewheel
+python -m pip install delvewheel wheel
```

Comment:
The `tools/wheels/repair_windows.sh` script has always needed `wheel` to [unpack the wheel so that the *.pyd* can be stripped](https://github.com/scipy/scipy/blob/ecaca3d4841cd1e297d3e803806d8ebae5c2c26f/tools/wheels/repair_windows.sh#L15). In the cp310, 311, 312 python builds `wheel` must have already been installed by something else (cibuildwheel, idk), I don't think we did. However, with cp313-dev, with no build isolation, `wheel` wasn't present, so the repair failed.
If you can strip the wheels during the build process then this could be avoided.

Reply:
Wheels should not need any stripping, that has to be a no-op since we moved to Meson. If not, that's a severe bug somewhere. We should be able to discard that whole step.

Reply:
Hmm, the reason is hidden somewhere in gh-16823 I think. Not sure if this was inherited from `multibuild` or not. The current comments about stripping in the `repair_windows.sh` don't ring a bell. But I do know that with `distutils` we had to strip by hand because there wasn't a way to change the default `-g` flag. In Meson we use `-Dbuildtype=release` which ensures debug info isn't added in the first place.

Reply:
This infra would've been added a long time after we transitioned to meson. I can't remember the history, apart from that comment. I've made a commit that's commented out that strip section to see what happens.

Reply:
Thanks. If it doesn't work, let's revert and add `wheel` - I'd be interested to investigate separately, but that doesn't have to hold up this PR.

Reply:
Now it works.